src/main/java/com/foo/gosucatcher/domain/item/domain/SubItem.java

```diff
@@ -9,40 +16,37 @@
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
 
-import com.foo.gosucatcher.global.BaseEntity;
-
-import lombok.AccessLevel;
-import lombok.Builder;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-
 @Getter
 @Entity
 @Table(name = "sub_items")
+@SQLDelete(sql = "UPDATE sub_item SET is_deleted = TRUE WHERE id = ?")
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class SubItem extends BaseEntity {
```

Comment:
```suggestion
@SQLDelete(sql = "UPDATE sub_items SET is_deleted = TRUE WHERE id = ?")
```
sub_items 테이블의 s가 빠진 것 같아요!!
그리고 삭제처리 된 데이터를 조회하지 않으려면 @Where(clause = "is_deleted = false")을 붙여주시면 좋을 것 같습니다ㅎㅎ